msteams-platform/resources/schema/manifest-schema.md

```diff
@@ -399,14 +399,14 @@ The value must be a valid HTML color code starting with '#', for example `#4464e
 
 **Optional** — array
 
-Used when your app experience has a team channel tab experience that requires extra configuration before it is added. Configurable tabs are supported only in the teams scope (not personal), and currently only **one** tab per app is supported.
+Used when your app experience has a team channel tab experience that requires extra configuration before it is added. Configurable tabs are supported only in the teams scope (not personal), and multiple tabs are supported per app.
```

Comment:
As per the [manifest schema v1.9](https://developer.microsoft.com/en-us/json-schemas/teams/v1.9/MicrosoftTeams.schema.json),  only one configurable tab is supported per app.  Could you please check and confirm?

![image](https://user-images.githubusercontent.com/31851992/115860726-8bd07c00-a44f-11eb-887b-51c2affbba64.png)